src/torrent.cpp

```diff
@@ -8283,7 +8277,7 @@ namespace libtorrent
 
 		int ret = 0;
 
-		std::int64_t const fin_time = finished_time();
+		std::int64_t const fin_time = duration_cast<seconds>(finished_time()).count();
```

Comment:
you can't have fields smaller than a byte (short of bitfields, but they're quite restricted, you can't define a time_point / duration type with smaller than one byte granularity).